README.md

```diff
@@ -723,6 +723,8 @@ Below are listed all the variables you can customize (you may also want to take
     # which will cause log lines to be written to "{{ openwisp2_path }}/log/openwisp2.log"
     # instead of "{{ openwisp2_path }}/log/celery.log" and "{{ openwisp2_path }}/log/celerybeat.log"
     openwisp2_django_celery_logging: false
+    # command to start celery from command line according to changes introduced from celery version 5.0 (Singularity)
+    command_to_start_celery : celery --app openwisp2 worker
```

Comment:
this command doesn't do anything, look into the supervisor configuration files, please double check `celery beat` as well.